app/views/admin/room_edit.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="container">
+  <div id="content-wrapper" class="d-flex flex-column">
+
+            <!-- Main Content -->
+            <div id="content">
+
+                <div class="container-fluid">
+
+                    <!-- Page Heading -->
+                    <h1 class="h3 mb-2 text-gray-800">Oda Düzenle</h1>
+                    
+
+                    <div class="card shadow mb-4">
+                        <div class="card-header py-3">
+                          <div class="row container">
+                            <div class="col-md-6" style="margin: auto;">
+                                <h6 class="m-0 font-weight-bold text-primary">Oda Bilgileri</h6>
+                                <%= "<div>#{@room.errors.full_messages.first}</div>".html_safe if @room.errors.any? %>   

```

Comment:
bununla neyi fixlediğini tam hatırlayamadım ama şu şekilde deneyebilirsin
```
<% if object.errors.any? %>
  <% object.errors.full_messages.each do |message| %>
     <%= message %> <br>
  <% end %>
<% end %> 
```

Bu dosyayı layout altına _errors.html.erb şeklinde oluşturursun.

Her bir model için tek tek aynı şeyleri yazmazsın da şöyle çağırırsın.
`<%= render 'layouts/errors' %>`

Reply:
DRY. Haklısın master.